easybuild/easyconfigs/s/Spyder/Spyder-5.0.5-foss-2020b.eb

```diff
@@ -0,0 +1,278 @@
+    }),
+    ('jinja2-time', '0.2.0', {
+        'checksums': ['d14eaa4d315e7688daa4969f616f226614350c48730bfa1692d2caebd8c90d40'],
+    }),
+    ('jupyterlab_pygments', '0.1.2', {
+        'checksums': ['cfcda0873626150932f438eccf0f8bf22bfa92345b814890ab360d666b254146'],
+    }),
+    ('importlib_metadata', '4.6.1', {
+        'checksums': ['079ada16b7fc30dfbb5d13399a5113110dab1aa7c2bc62f66af75f0b717c8cac'],
+    }),
+    ('keyring', '21.8.0', {
+        'preinstallopts': """sed -i 's/setup()/setup(version="%(version)s")/g' setup.py && """,
+        'checksums': ['1746d3ac913d449a090caf11e9e4af00e26c3f7f7e81027872192b2398b98675'],
+    }),
+    ('keyrings.alt', '4.1.0', {
+        'preinstallopts': """sed -i 's/setup()/setup(version="%(version)s")/g' setup.py && """,
+        'checksums': ['52ccb61d6f16c10f32f30d38cceef7811ed48e086d73e3bae86f0854352c4ab2'],
+    }),
+    ('lazy-object-proxy', '1.6.0', {
+        'checksums': ['489000d368377571c6f982fba6497f2aa13c6d1facc40660963da62f5c379726'],
+    }),
+    ('mccabe', '0.6.1', {
+        'checksums': ['dd8d182285a0fe56bace7f45b5e7d1a6ebcbf524e8f3bd87eb0f125271b8831f'],
+    }),
+    ('jupyter_client', '6.1.12', {
+        'checksums': ['c4bca1d0846186ca8be97f4d2fa6d2bae889cce4892a167ffa1ba6bd1f73e782'],
+    }),
+    ('nbclient', '0.5.3', {
+        'checksums': ['db17271330c68c8c88d46d72349e24c147bb6f34ec82d8481a8f025c4d26589c'],
+    }),
+    ('nest_asyncio', '1.5.1', {
+        'checksums': ['afc5a1c515210a23c461932765691ad39e8eba6551c055ac8d5546e69250d0aa'],
+    }),
+    ('numpydoc', '1.1.0', {
+        'checksums': ['c36fd6cb7ffdc9b4e165a43f67bf6271a7b024d0bb6b00ac468c9e2bfc76448e'],
+    }),
+    ('pathtools', '0.1.2', {
+        'checksums': ['7c35c5421a39bb82e58018febd90e3b6e5db34c5443aaaf742b3f33d4655f1c0'],
+    }),
+    ('poyo', '0.5.0', {
+        'checksums': ['e26956aa780c45f011ca9886f044590e2d8fd8b61db7b1c1cf4e0869f48ed4dd'],
+    }),
+    ('pycodestyle', '2.7.0', {
+        'checksums': ['c389c1d06bf7904078ca03399a4816f974a1d590090fecea0c63ec26ebaf1cef'],
+    }),
+    ('pydocstyle', '6.1.1', {
+        'checksums': ['1d41b7c459ba0ee6c345f2eb9ae827cab14a7533a88c5c6f7e94923f72df92dc'],
+    }),
+    ('pyflakes', '2.3.1', {
+        'checksums': ['f5bc8ecabc05bb9d291eb5203d6810b49040f6ff446a756326104746cc00c1db'],
+    }),
+    ('isort', '5.9.2', {
+        'checksums': ['f65ce5bd4cbc6abdfbe29afc2f0245538ab358c14590912df638033f157d555e'],
+    }),
+    ('pylint', '2.9.5', {
+        'checksums': ['1f333dc72ef7f5ea166b3230936ebcfb1f3b722e76c980cb9fe6b9f95e8d3172'],
+    }),
+    ('ujson', '4.0.2', {
+        'checksums': ['c615a9e9e378a7383b756b7e7a73c38b22aeb8967a8bfbffd4741f7ffd043c4d'],
+    }),
+    ('python-language-server', '0.36.2', {
+        'modulename': 'pyls',
+        'checksums': ['9984c84a67ee2c5102c8e703215f407fcfa5e62b0ae86c9572d0ada8c4b417b0'],
+    }),
+    ('pyrsistent', '0.18.0', {
```

Comment:
Already included in deps (slightly different version, but maybe still compatible)